model/plugins/create_role_mapping/operations.smithy

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: Apache-2.0
+//
+//  The OpenSearch Contributors require contributions made to
+//  this file be licensed under the Apache-2.0 license or a
+//  compatible open source license.
+
+$version: "2"
+namespace OpenSearch
+use opensearch.openapi#vendorExtensions
+
+@externalDocumentation(
+    "API Reference": "https://opensearch.org/docs/2.7/security/access-control/api/#create-role-mapping"
+)
+
+@vendorExtensions(
+    "x-operation-group": "create_role_mapping",
```

Comment:
operation group should be security.create_role_mapping. We must use the security namespace for every other security operations